src/lib/abacus/dydxChainTransactions.ts

```diff
@@ -272,30 +282,24 @@ class DydxChainTransactions implements AbacusDYDXChainTransactionsProtocol {
       throw new Error('Missing compositeClient or localWallet');
     }
 
-    const {
-      orderId,
-      subaccountNumber,
-      clientId,
-      orderFlags,
-      clobPairId,
-      goodTilBlock,
-      goodTilBlockTime,
-    } = params ?? {};
+    const { subaccountNumber, clientId, orderFlags, clobPairId, goodTilBlock, goodTilBlockTime } =
+      params ?? {};
 
     try {
       const tx = await this.compositeClient?.cancelRawOrder(
         new SubaccountClient(this.localWallet, subaccountNumber),
         clientId,
         orderFlags,
         clobPairId,
-        goodTilBlock || undefined,
-        goodTilBlockTime || undefined
+        goodTilBlock === 0 ? undefined : goodTilBlock ?? undefined,
```

Comment:
mega nit: I wonder if this is less readable than just using eslint disable

Reply:
Yeah I was experimenting here. I don't have a strong opinion other than being in favor of being explicit about using truthiness.